.php_cs.dist

```diff
@@ -15,18 +15,20 @@ $config
         'array_syntax' => array('syntax' => 'long'),
         'binary_operator_spaces' => false,
         'concat_space' => array('spacing' => 'one'),
+        'increment_style' => false,
+        'no_alias_functions' => true,
         'no_useless_else' => true,
         'no_useless_return' => true,
         'ordered_imports' => true,
         'phpdoc_no_package' => false,
         'phpdoc_order' => true,
         'phpdoc_summary' => false,
         'pre_increment' => false,
-        'trailing_comma_in_multiline_array' => false,
         'simplified_null_return' => false,
+        'trailing_comma_in_multiline_array' => false,
         'yoda_style' => null,
-        'increment_style' => false,
     ))
+    ->setRiskyAllowed(true)
```

Comment:
I find this concerning - there have been a number of times in the past when php-cs-fixer has shipped changes that resulted in it altering the execution logic of the project code, rather than making purely cosmetic changes. Thankfully, this caused the testsuite to fail, so nothing broken made it through review, but still. Anything that php-cs-fixer labels risky, or has the potential to change execution logic, worries me.

Reply:
To clarify - php-cs-fixer has altered logic with *no warning, and no changes in configuration*. This behaviour is IMO unacceptable. I don't have an issue with the join -> implode changes it's made this time, but if we're going to commit this, I'd like some certainty it won't make changes in the future that break things (e.g. by replacing deprecated function names that are required for PHP 5.3 compatibility).

Reply:
That's why I would suggest to pin `friendsofphp/php-cs-fixer` to at least a specific minor version, what do you think?

Reply:
I think pinning is a very good idea. It doesn't entirely solve the issue, but it will help.

@mathroc - php-cs-fixer is yours, how do you feel about this?